MikuMikuModel/Nodes/Sprites/SpriteNode.cs

```diff
@@ -161,6 +163,33 @@ protected override void Initialize()
                 using ( var bitmap = SpriteCropper.Crop( Data, FindParent<SpriteSetNode>().Data ) )
                     bitmap.Save( filePath, imageFormat );
             } );
+
+            AddReplaceHandler<Bitmap>( filepath =>
+            {
+                var texture = FindParent<SpriteSetNode>().Data.TextureSet.Textures[ ( int ) Data.TextureIndex ];
+                var bitmap = TextureDecoder.DecodeToBitmap( texture );
+                Bitmap newBitmap;
+                if ( texture.IsYCbCr )
+                    newBitmap = TextureDecoder.DecodeToBitmap( TextureEncoder.EncodeYCbCrFromFile( filepath ) );
+                else
+                    newBitmap = TextureDecoder.DecodeToBitmap( TextureEncoder.EncodeFromFile( filepath, texture.Format, texture.UsesMipMaps ) );
+                
```

Comment:
This bit is unnecessary. You can directly use bitmap's filepath constructor to load the image.

Reply:
I really shouldnt be allowed to write code after this